mitiq/pec/channels.py

```diff
@@ -106,20 +106,16 @@ def _operation_to_choi(operation_tree: OP_TREE) -> np.ndarray:
     return _circuit_to_choi(circuit)
 
 
-def _tensor_product_kraus(
-    local_kraus: List[np.ndarray], num_channels: int
-) -> List[np.ndarray]:
-    """Given the a local channel (defined by its kraus operators),
-    this functions returns the Kraus operators corresponding to tensor
-    product of "num_channels" copies of the input channel.
+def tensor_product(*args: np.ndarray) -> np.ndarray:
+    """Returns the Kronecker product of the input arguments (NumPy objects).
```

Comment:
```suggestion
    """Returns the Kronecker product of the input arguments (NumPy arrays).
```

Reply:
I guess really it doesn't have to be arrays or numpy types. e.g., `np.kron` for scalars is multiplication. `np.kron(a, b)` says `a` and `b` should be "array like".

Reply:
changed with `product of the input array-like arguments.`